dali/dfu/dfurun.cpp

```diff
@@ -533,6 +533,24 @@ class CDFUengine: public CInterface, implements IDFUengine
         }
     }
 
+    void ensureAFilePermissions(IFileDescriptor * fd, SecAccessFlags perm, bool write)
```

Comment:
trivial: not sure about the 'A', I think "ensureFilePermissions" would be better.